docs/general-development/working-with-tasks-in-sharepoint-workflows-using-visual-studio-2012.md

```diff
@@ -1,23 +1,23 @@
 ---
-title: Working with Tasks in SharePoint Workflows using Visual Studio 2012
-description: Learn about the new and revised workflow task framework that was introduced in SharePoint, which is built on the new Workflow Manager.
-ms.date: 10/15/2022
+title: Working with Tasks in SharePoint 2013 Platform Workflows using Visual Studio

```

Comment:
Can you remove 'Platform'? The official name isn't "SharePoint 2023 Platform". 

Also, it looks like '2013' was added in lots of places after SharePoint. When referring to "SharePoint 2013 Workflows", that's fine, but otherwise it's "SharePoint Server 2013".

Can you update the PR to account for this?